examples/taskmaster.yaml

```
@@ -0,0 +1,154 @@
+tests:
+  - project add test1: assert result.exit_code == 0
+  - project list: assert "test1" in result.output
+  - project add test2: assert result.exit_code == 0
+  - project list: assert "test1" in result.output and "test2" in result.output
+  - task add test1 task1: assert result.exit_code == 0
+  - task add test1 task2: assert result.exit_code == 0
+  - task list test1: assert "task1" in result.output and "task2" in result.output
+  - task remove test1 task1: assert "removed" in result.output
+  - project remove test1: assert "removed" in result.output
+  - task list test1: assert "No tasks found" in result.output
+  - $ rm tasks.json
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)